docker-run.sh

```diff
@@ -1,3 +1,15 @@
 #!/bin/bash
 
-docker run --name localproxy --rm -it aws-iot-securetunneling-localproxy:latest bash
+while getopts p: flag
+do
+    case "${flag}" in
+        p) port=${OPTARG};;
+    esac
+done
+
+if [ -z $port ]; then
+	docker run --name localproxy --rm -it aws-iot-securetunneling-localproxy:latest bash;
+else
+	docker run --name localproxy -expose=$port -p $port:$port --rm -it aws-iot-securetunneling-localproxy:latest bash;
```

Comment:
`-expose` should be `--expose` (tested on my machine that `-expose` fails)

Reply:
Not sure why is it not failing on my my machine but you're right, fixed it.